docs/users/linea-voyage/lxp/index.mdx

```diff
@@ -144,6 +144,6 @@ To start the Account binding process, follow the steps below:
 
 :::info[What if I want to bind an account from a different Secret Recovery Phrase/seed phrase?]
 
-Refer to the support guide [here](https://support.linea.build/hc/en-us/articles/21459048100123).
+Refer to the support guide [here](https://support.linea.build/linea-voyage/lxp-binding-an-account-from-a-different-secret-recovery-phrase-).
```

Comment:
```suggestion
Refer to the [support guide](https://support.linea.build/linea-voyage/lxp-binding-an-account-from-a-different-secret-recovery-phrase-).
```

Reply:
There are so many instances of this that would require refactoring sentence to fix -- is Linea not following MS style or Google dev style guide? wen Vale this should be flagged, but only relevant if they follow same guides